code/Test_definitions/Geofencing.feature

```diff
@@ -0,0 +1,114 @@
+
+
+@Geofencing 
+Feature: Camara Geofencing Subscriptions API ,0.2.0-rc3 Operations on subscriptions 
+
+# Input to be provided by the implementation to the tests
+# References to OAS spec schemas refer to schemas specified in geofencing-subscriptions.yaml, version v0.2.0-rc3
+
+  Background: Common Geofencing Subscriptions setup
+    Given the resource "/geofencing-subscriptions/v0/subscriptions"  as geofencing Url                                                            |
+    And the header "Content-Type" is set to "application/json"
+    And the header "Authorization" is set to a valid access token
+    And the header "x-correlator" is set to a UUID value
+    And the request body is set by default to a request body compliant with the schema
+
+  @geofencing_subscriptions_01_Create_geofencing_subscription_for_a_device
+  Scenario:  Create geofencing subscription
+    Given they use the geofencing url
+    When they create subscription
+    Then Response code is 201
+
+  @geofencing_subscriptions_02_Operation_to_retrieve_list_of_subscriptions
+  Scenario: Get a list of subscriptions.
+    Given they use the geofencing url
+    When they get all subscriptions
+    Then Response code is 200
+
+
+  @geofencing_subscriptions_03_Operation_to_retrieve_subscription_based_on_the_provided_ID
+  Scenario: Get a subscription based on provided id.
+    Given they use the geofencing url
+    When they get subscription for subscription-id
+    Then Response code is 200
+
+  @geofencing_subscriptions_04_Operation_to_delete_subscription_based_on_the_provided_ID
+  Scenario: Delete a subscription based on provided id.
+    Given they use the geofencing url
+    When they delete subscription for subscription-id
+    Then Response code is 204
+
+  @geofencing_subscriptions_05_Create_invalid_geofencing_subscription_for_a_device
+  Scenario:  Create geofencing subscription with invalid parameter
+    Given they use the geofencing url
+    When they create subscription with invalid parameter
+    Then Response code is 400
+
+  @geofencing_subscriptions_06_Get_invalid_geofencing_subscription_for_a_device
```

Comment:
I know I gave the same comment before :) but here we better say the subscription is "unknown" to the system. "ınvalid" may be understood as invalid format. For invalid format, we now have test #15.